src/test/raii_event_tests.cpp

```diff
@@ -4,8 +4,18 @@
 
 #include <event2/event.h>
 
+#include <boost/test/unit_test.hpp>
+
+BOOST_FIXTURE_TEST_SUITE(raii_event_tests, BasicTestingSetup)
+
+#ifndef EVENT_SET_MEM_FUNCTIONS_IMPLEMENTED
+BOOST_AUTO_TEST_CASE(raii_event_creation)
+{
+    // dummy; do nothing
```

Comment:
Suggest replacing with:

```C++
    // It would probably be ideal to report skipped, but boost::test doesn't seem to make that practical (at least not in versions available with common distros)
    BOOST_TEST_MESSAGE("Skipping raii_event_tess: libevent doesn't support event_set_mem_functions");
```